dist/image-sequencer.js

```diff
@@ -47312,3374 +47312,3376 @@ arguments[4][38][0].apply(exports,arguments)
-module.exports={
-    "name": "Average",
-    "description": "Average all pixel color",
-    "inputs": {
-    }
-}
-
-},{}],149:[function(require,module,exports){
-module.exports = function Dynamic(options, UI, util) {
-
-  options.func = options.func || "function(r1, g1, b1, a1, r2, g2, b2, a2) { return [ r1, g2, b2, a2 ] }";
-
-  var output;
-
-  // This function is called on every draw.
-  function draw(input, callback, progressObj) {
-
-    progressObj.stop(true);
-    progressObj.overrideFlag = true;
-
-    var step = this;
-
-    // convert to runnable code:
-    if (typeof options.func === "string") eval('options.func = ' + options.func);
-
-    var getPixels = require('get-pixels');
-
-    // save first image's pixels
-    var priorStep = this.getStep(-2);
-
-    getPixels(priorStep.output.src, function(err, pixels) {
-      options.firstImagePixels = pixels;
-
-      function changePixel(r2, g2, b2, a2, x, y) {
-        // blend!
-        var p = options.firstImagePixels;
-        return options.func(
-          r2, g2, b2, a2,
-          p.get(x, y, 0),
-          p.get(x, y, 1),
-          p.get(x, y, 2),
-          p.get(x, y, 3)
-        )
-      }
-
-      function output(image, datauri, mimetype) {
-
-        // This output is accessible by Image Sequencer
-        step.output = { src: datauri, format: mimetype };
-
-      }
-
-      // run PixelManipulatin on second image's pixels
-      return require('../_nomodule/PixelManipulation.js')(input, {
-        output: output,
-        changePixel: changePixel,
-        format: input.format,
-        image: options.image,
-        inBrowser: options.inBrowser,
-        callback: callback
-      });
-    });
-  }
-
-  return {
-    options: options,
-    draw: draw,
-    output: output,
-    UI: UI
-  }
-}
-
-},{"../_nomodule/PixelManipulation.js":207,"get-pixels":23}],150:[function(require,module,exports){
-arguments[4][147][0].apply(exports,arguments)
-},{"./Module":149,"./info.json":151,"dup":147}],151:[function(require,module,exports){
-module.exports={
-  "name": "Blend",
-  "description": "Blend the past two image steps with the given function. Defaults to using the red channel from image 1 and the green and blue and alpha channels of image 2. Easier to use interfaces coming soon!",
-  "inputs": {
-    "blend": {
-      "type": "input",
-      "desc": "Function to use to blend the two images.",
-      "default": "function(r1, g1, b1, a1, r2, g2, b2, a2) { return [ r1, g2, b2, a2 ] }"
-    }
-  }
-}
-
-},{}],152:[function(require,module,exports){
-module.exports = exports = function(pixels, blur) {
-    let kernel = kernelGenerator(blur, 1), oldpix = pixels;
-    kernel = flipKernel(kernel);
-
-    for (let i = 0; i < pixels.shape[0]; i++) {
-        for (let j = 0; j < pixels.shape[1]; j++) {
-            let neighboutPos = getNeighbouringPixelPositions([i, j]);
-            let acc = [0.0, 0.0, 0.0, 0.0];
-            for 
```

Comment:
Here it is!!!

Reply:
I found this!!
![ss_5](https://user-images.githubusercontent.com/21158457/45388421-89220f00-b636-11e8-9d9d-55216f9db2cd.jpg)

Reply:
This code is also getting displayed in additions.
![ss_6](https://user-images.githubusercontent.com/21158457/45388726-70febf80-b637-11e8-9fa1-201d32e03d31.jpg)